app/src/lib/Level.svelte

```diff
@@ -1,9 +1,17 @@
 <script>
 	/** @type {Concept['level']} */
 	export let level
+
+	const level_descriptions = [
+		'Semantic Primitive',
+		'Semantic Molecule',
+		'Complex with an Insertion Rule',
+		'Complex without an Insertion Rule',
+		'Inexplicable'
```

Comment:
you can drop and extra `,` at the end of that last entry to help keep diff's clean in the future.